metadata-db-app/src/cmr/metadata_db/services/generic_documents.clj

```diff
@@ -57,7 +57,7 @@
                             :concept-type concept-type
                             :document-name (subs doc-name 0 (min 20 (count doc-name)))
                             :schema concept-type
-                            :format (identity concept-type)
+                            :format (str "application/vnd.nasa.cmr.umm+json;version=" version)
```

Comment:
Do generics only support this mime type?

Reply:
right now, we only ingest UMM_JSON. Just like subscriptions, tools, variables, etc.